src/main/java/io/hops/metadata/ndb/ClusterjConnector.java

```diff
@@ -34,6 +34,7 @@
 import io.hops.metadata.ndb.mysqlserver.MysqlServerConnector;
 import io.hops.metadata.ndb.wrapper.HopsSession;
 import io.hops.metadata.ndb.wrapper.HopsTransaction;
+import io.hops.metadata.yarn.dal.AppProvenanceDataAccess;
```

Comment:
It seems you missed to import the other two classes 
FileProvenanceDataAccess  and FileProvXAttrBufferDataAccess

Reply:
there is an import for all the hdfs DALs

https://github.com/hopshadoop/hops-metadata-dal-impl-ndb/blob/acada8c148133fff5c8df49a5c341aadd8d4e86e/src/main/java/io/hops/metadata/ndb/ClusterjConnector.java#L30

Reply:
import io.hops.metadata.hdfs.dal.*;